solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -1107,8 +1100,7 @@ contract StandardPoolConverter is ConverterVersion, IConverter, TokenHandler, To
 
         for (uint256 i = 0; i < _reserveTokens.length; i++) {
```

Comment:
it's time to merge the add liquidity to empty/non empty pool functions - they are basically the same function, the only difference is the calculated balance - this is where we should have the differentiation.